addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -9,7 +9,7 @@
                         <button class="btn btn-link o_ChatterTopbar_button o_ChatterTopbar_buttonSendMessage"
                             type="button"
                             t-att-class="{
-                                'o-active': chatter.isComposerVisible and !chatter.isComposerLog,
+                                'o-active': chatter.isComposerVisible and (!chatter.composer or !chatter.composer.isLog),
```

Comment:
condition here is not correct, it should be `chatter.composer` (not negative).

But do we need it? Feels like `isComposerVisible` should always be falsy if composer doesn't exist.

Reply:
Indeed, should be `chatter.isComposerVisible and chatter.composer and !chatter.composer.isLog`.
`isComposerVisible` is a UI concern, while `composer` field is always set when there is a thread, so it's relevant to have both parts in conditional expression.

Reply:
I think indeed chatter.composer would always be defined here so negative or not would not change a lot

Reply:
Just write the condition Alex said to be safe, then we can merge.